inst/template/default.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html{{ if (isTRUE(nzchar(lang))) paste0("lang=\"", lang, "\"") }}>
```

Comment:
Just checking: if `lang` is NULL, would any extra spaces be inserted here?

Reply:
Nope

```
❯❯ isTRUE(nzchar(NULL))
[1] FALSE
```

Reply:
And everything between the brackets (and including the brackets) will be removed

Reply:
Sounds good - I was mostly interested in making sure that the `{{ }}` didn't result in any extra spaces.